docs/tutorials/web.md

```diff
@@ -5,10 +5,405 @@ has_children: false
+  // 1. Convert image to tensor
+  const imageTensor = await getImageTensorFromPath(path);
+  // 2. Run model
+  const [predictions, inferenceTime] = await runSqueezenetModel(imageTensor);
+  // 3. Return predictions and the amount of time it took to inference.
+  return [predictions, inferenceTime];
+}
+```
+### imageHelper.ts
+
+First, we need to get our image from path (can be local or url) and convert it to a tensor. The `getImageTensorFromPath` function in the `imageHelper.ts` uses `JIMP` to read the file, resize and return the `imageData`. [JIMP](https://www.npmjs.com/package/jimp) is a JavaScript image manipulation library. It has many built in functions for working with image data such as resizing, grey scale, write, and more. In this example we only need to resize however in your code you may need additional image data processing. 
+
+```javascript
+import Jimp  from 'jimp/*';
+import { Tensor } from 'onnxruntime-web';
+const jimp = require('jimp');
+
+export default async function getImageTensorFromPath(path: string, dims: number[] =  [1, 3, 224, 224]): Promise<Tensor> {
+  // 1. load the image  
+  var image = await loadImagefromPath(path, dims[2], dims[3]);
+  // 2. convert to tensor
+  var imageTensor = imageDataToTensor(image, dims);
+  // 3. return the tensor
+  return imageTensor;
+}
+
+async function loadImagefromPath(path: string, width: number = 224, height: number= 224): Promise<Jimp> {
+ // Use Jimp to load the image and resize it.
+    var imageData = await jimp.default.read(path).then((imageBuffer: Jimp) => {
+        return imageBuffer.resize(width, height)
+      });
+
+    return imageData;
+}
+```
+
+Once we have the imageData we will send it into the `imageDataToTensor` function to convert it to an ORT Tensor for inferencing. To convert an image to a tensor in JavaScript we need to get the RGB (Red, Green, Blue) values into arrays. To do this we will loop through the `imageBufferData` by each pixels 4 channels of RGBA. Once we have the RGB pixel channels for the image, then we create the `Float32Array` from the `transposedData` and divide by 
+255 to normalize the value. Why does 255 normalize the pixel value? Well normalization is a technique used to change values to a common scale without distorting the differences. 255 is the max number for an RGB value, so dividing by 255 normalizes our values to between 0 and 1 without losing the statistical differences.  Now that we have the `Float32Array` representation of the image we can create the ORT Tensor by sending in the type, data, and dimensions. Then we return the inputTensor for inferencing.
+
+```javascript
+function imageDataToTensor(image: Jimp, dims: number[]): Tensor {
+    // 1. Get buffer data from image and create R, G, and B arrays.
+    var imageBufferData = image.bitmap.data;
+    const [redArray, greenArray, blueArray] = new Array(new Array<number>(), Array<number>(), Array<number>());
```

Comment:
```
const redArray = [];
const greenArray = [];
const blueArray = [];
```

Reply:
Both of these work and its really more of a preference, so I think we can leave it.

Reply:
the current code does not keep consistent code style, (like using `new Array()` and `Array()` at the same time), and is less verbose.

if you prefer to use `Array`, it is still better to do like:
`const [redArray, greenArray, blueArray] = [new Array<number>(), new Array<number>(), new Array<number>()]`

Reply:
I see, added new to the other two array objects.